src/PowerFlowData.jl

```diff
@@ -15,6 +15,34 @@ abstract type SystemPowerFlowContainer <: PowerFlowContainer end
 
 get_system(container::SystemPowerFlowContainer) = container.system
 
+struct LCCParameters
+    arc::Vector{PSY.Arc}
+    p_set::Matrix{Float64}
+
+    rectifier_i_dc::Matrix{Float64}
+    inverter_i_dc::Matrix{Float64}
+
+    dc_line_resistance::Vector{Float64}
+
+    rectifier_tap::Matrix{Float64}
+    inverter_tap::Matrix{Float64}
+
+    rectifier_delay_angle::Matrix{Float64}     # α
+    inverter_extinction_angle::Matrix{Float64} # γ
+
+    rectifier_phi::Matrix{Float64}
```

Comment:
This is okay for me, I'll just note that sometimes we use special characters (search PSY codebase for `α`, `γ`, see also PF `robust_homotopy_method.jl`), we should probably be consistent about whether we use special characters or their names.